spec/game_spec.rb

```diff
@@ -1,260 +0,0 @@
-require 'spec_helper'
```

Comment:
Why did you delete all of the game specs? Don't you test `play` and a few other functions in `Game`?

Reply:
I was confused, I thought all of the methods were being tested with each of the unit tests in their respective class specs. I talked to Josh about integration tests today and I reimplemented them.